README.md

```diff
@@ -88,6 +88,27 @@ You can build the local project `docker-compose` on Mac or Linux,  or into a dis
     + On Linux, please also follow the [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/).
 + System Python (that is: no version manager, such as pyenv, should be active).
 
+# Running Lumigator
+
+## Running Lumigator Locally
+
+1. `git clone git@github.com:mozilla-ai/lumigator.git`
+2. `make start-lumigator`
+3. The application should be available at http://localhost:8000. 
```

Comment:
Perhaps "The REST API should be available..."? To me application looks a bit more advanced than our API UI, but it can be subjective. WDYT?